index.js

```diff
@@ -3,7 +3,7 @@
 var BufferStreams = require('bufferstreams');
 var htmlmin = require('html-minifier');
 var gutil = require('gulp-util');
-var objectAssign = require('object-assign');
```

Comment:
> You're looking at it from your perspective

No. As its name suggests, "Using gulp-html with `gulp`" is just a general use case of this plugin.

I'd like to ask you when you use gulp-html without `gulp`, or even `vinyl-fs`.